src/JuxtaposeApplication.jsx

```diff
@@ -48,7 +48,7 @@ export default class JuxtaposeApplication extends React.Component {
                    const source = extractSource(assetCtx.sources);
                    const annotation = extractAnnotation(assetCtx, assetData.annotationId);
                    
-                   if (e.detail.caller.type === 'spine') {
+                   if (e.detail.caller === 'spine') {
```

Comment:
See https://github.com/ccnmtl/juxtapose/blob/master/src/MediaTrack.jsx#L12. It should be coming through as an object, not a string.